Tests/Unit/Microsoft365DSC/Microsoft365DSC.EXOReportSubmissionPolicy.Tests.ps1

```diff
@@ -0,0 +1,267 @@
+                    ReportJunkToCustomizedAddress          = $False;
+                    ReportNotJunkToCustomizedAddress       = $False;
+                    ReportPhishToCustomizedAddress         = $False;
+                }
+
+                Mock -CommandName Get-ReportSubmissionPolicy -MockWith {
+                    return @{
+                        Ensure                                 = 'Present'
+                        Credential                             = $Credential
+                        IsSingleInstance                       = 'Yes';
+                        Identity                               = "DefaultReportSubmissionPolicy"
+                        DisableQuarantineReportingOption       = $False;
+                        EnableCustomNotificationSender         = $False;
+                        EnableOrganizationBranding             = $False;
+                        EnableReportToMicrosoft                = $False;
+                        EnableThirdPartyAddress                = $False;
+                        EnableUserEmailNotification            = $False;
+                        PostSubmitMessageEnabled               = $False;
+                        PreSubmitMessageEnabled                = $False;
+                        ReportJunkToCustomizedAddress          = $False;
+                        ReportNotJunkToCustomizedAddress       = $False;
+                        ReportPhishToCustomizedAddress         = $False;
+                    }
+                }
+
+                Mock -CommandName Set-ReportSubmissionPolicy -MockWith {
+                    return @{
+
+                    }
+                }
+            }
+
+            It 'Should return false from the Test method' {
+                Test-TargetResource @testParams | Should -Be $false
+            }
+
+            It 'Should Successfully call the Set method' {
+                Set-TargetResource @testParams
+            }
+        }
+
+        Context -Name 'ReportSubmissionPolicy removal.' -Fixture {
+            BeforeAll {
+                $testParams = @{
+                    Ensure              = 'Absent'
+                    Credential          = $Credential
+                    Identity            = 'DefaultReportSubmissionPolicy'
+                    IsSingleInstance    = 'Yes';
+                }
+
+                Mock -CommandName Get-ReportSubmissionPolicy -MockWith {
+                    return @{
+                        Identity = 'DefaultReportSubmissionPolicy'
+                    }
+                }
+
+                Mock -CommandName Remove-ReportSubmissionPolicy -MockWith {
+                    return @{
+
+                    }
+                }
+            }
+
+            It 'Should return false from the Test method' {
+                Test-TargetResource @testParams | Should -Be $false
+            }
+
+            It 'Should Remove the Policy in the Set method' {
+                Set-TargetResource @testParams
```

Comment:
Same comment as the others, but in this case the Remove-ReportSubmissionPolicy cmdlet.